dev/Expander/Expander_themeresources.xaml

```diff
@@ -27,23 +31,28 @@
             <StaticResource x:Key="ExpanderChevronBorderBrush" ResourceKey="SubtleFillColorTransparentBrush" />
             <StaticResource x:Key="ExpanderChevronBorderPointerOverBrush" ResourceKey="SubtleFillColorTransparentBrush" />
             <StaticResource x:Key="ExpanderChevronBorderPressedBrush" ResourceKey="SubtleFillColorTransparentBrush" />
-            <StaticResource x:Key="ExpanderDisabledForeground" ResourceKey="TextFillColorDisabledBrush" />
-            <StaticResource x:Key="ExpanderDisabledBorderBrush" ResourceKey="ExpanderBorderBrush" />
-            <Thickness x:Key="ExpanderBorderThickness">1</Thickness>
             <Thickness x:Key="ExpanderChevronBorderThickness">0</Thickness>
+
+            <!-- Content -->
+            <StaticResource x:Key="ExpanderContentBackground" ResourceKey="CardBackgroundFillColorSecondaryBrush" />
+            <StaticResource x:Key="ExpanderContentBorderBrush" ResourceKey="ExpanderHeaderBorderBrush" />
```

Comment:
Have you tested that this works? I'm worried that if you override the ExpanderHeaderBorderBrush at the local scope the ExpanderContentBorderBrush wont see your override.  This is because the resource lookup doesn't restart when a resource redirection is encountered. So when the template is expanded and this resource is found, we'll start looking for a value for the ExpanderHeaderBorderBrush from our current position in the tree walk, which will miss the local overrides lower in the tree.

Reply:
> Have you tested that this works

No, I have not tested this specifically and didn't have a reason to at the time. My changes here were only renaming and I copied the original functionality. Code before renaming is below:

https://github.com/microsoft/microsoft-ui-xaml/blob/d20fcb84ea3cc5e21a82fd4f44547471e7259078/dev/Expander/Expander_themeresources.xaml#L41

https://github.com/microsoft/microsoft-ui-xaml/blob/d20fcb84ea3cc5e21a82fd4f44547471e7259078/dev/Expander/Expander_themeresources.xaml#L45-L46

`ExpanderBorderBrush` renamed to `ExpanderHeaderBorderBrush`
`ExpanderDropDownBorderBrush` renamed to `ExpanderContentBorderBrush`

Resource resolution is tricky to understand for those outside the codebase and is not well documented at least that I've found. So I'll take your word for how it functions in this area. If you want me to make the change that's fine. It will just be set to `CardStrokeColorDefaultBrush` as well.

Reply:
Note `ExpanderHeaderDisabledBorderBrush` is also affected but would make sense to follow the `ExpanderHeaderBorderBrush` when changed. Still, this should change as well due to potential resource lookup issues.

https://github.com/microsoft/microsoft-ui-xaml/blob/cf8bcef4058f1b4f605fffb01504ad99ac25d03c/dev/Expander/Expander_themeresources.xaml#L22